lib/ruby_lsp/tapioca/run_gem_rbi_check.rb

```diff
@@ -81,15 +79,16 @@ def generate_gem_rbis
       #: (Array[String] gems) -> void
       def execute_tapioca_gem_command(gems)
         Bundler.with_unbundled_env do
-          stdout, stderr, status = T.unsafe(Open3).capture3(
-            "bundle",
-            "exec",
-            "tapioca",
-            "gem",
-            "--lsp_addon",
-            *gems,
-            chdir: project_path,
-          )
+          stdout, stderr, status = Open3 #: as untyped
+            .capture3(
+              "bundle",
+              "exec",
+              "tapioca",
+              "gem",
+              "--lsp_addon",
+              *gems,
```

Comment:
What do you think of ignoring `7019` error in this project as well?

Reply:
I think we should, but not as part of this PR. We haven't migrated the entire of Tapioca to use RBS comments - this is just for the add-on.

As part of the full migration, we should then turn that error off.